main.py

```diff
@@ -0,0 +1,64 @@
+import discord
+import os
+from discord.ext import commands
+from secrettoken import token
+from discord.ext.commands import has_permissions, MissingPermissions
+from discord.utils import get
+
+intents = discord.Intents.default()
+intents.presences = True
+intents.members = True
+ 
+client = commands.Bot(command_prefix='$', intents=intents)
+
+@client.command()
+async def load(ctx, extension):
+    client.load_extension(f'cogs.{extension}')
+
+@client.command()
+async def unload(ctx, extension):
+    client.unload_extension(f'cogs.{extension}')
+
+for file in os.listdir('./cogs'):
+    if file.endswith(".py"):
```

Comment:
@stephenlee91 @thebosshippo Do we like this check? I'm thinking .py files could potentially be put under here without being a cog on its own. Perhaps we have a global list of what cogs we have and we check if the file is in that list instead? Any thoughts

Reply:
Not too familiar with cogs. Are you saying we check if the .py file is in the list of cogs? Not too sure what the second sentence meant. I think that if the cogs take up too many lines then yes it would be better to have a cogs directory with .py files inside. If the cogs are short and simple then it would be better to group them together in a file.

Reply:
Looking at Benji's autoassign.py, I think it's better we have individual .py files for each cog because I think down the line we will be adding a lot more functions to the bot and we don't know how long the files will be. Therefore it is better to anticipate for long files. I also think it's better to add files to the directory instead of updating one file everytime we want to add a function.

Reply:
That's true. However, based on this code, if we happen to have a non-cog .py file in the .../cogs folder, it will be treated as a cog. That's the problem that I'm seeing that could occur. Otherwise I agree on the structure of having a cog per file.